doc/source/development/contributing_codebase.rst

```diff
@@ -338,7 +338,22 @@ Writing tests
 
 All tests should go into the ``tests`` subdirectory of the specific package.
 This folder contains many current examples of tests, and we suggest looking to these for
-inspiration. Ideally, there should be one, and only one, obvious place for a test to reside.
+inspiration.
+
+As a general tip, you can use the search functionality in your integrated development
+environment (IDE) or the grep command in a terminal to find test files in which the method
+is called. If you are unsure of the best location to put your test, take your best guess,
+but note that reviewers may request that you move the test to a different location.
+
+To use grep, you can run the following command in a terminal:
+
+``grep -r "function_name("``
```

Comment:
should we suggest `git grep` instead? because then:
- it'll be cross-platform
- it'll only search in tracked files
- it doesn't require `-r`

Reply:
Updated, and learnt about git grep, thanks!